<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>References &mdash; PlantPhoneDB-Tutorial v1.0.0 documentation</title>
      <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
      <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
        <script src="_static/jquery.js"></script>
        <script src="_static/underscore.js"></script>
        <script src="_static/doctools.js"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Document" href="06-Document.html" />
    <link rel="prev" title="Download" href="04-Download.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
            <a href="index.html" class="icon icon-home"> PlantPhoneDB-Tutorial
          </a>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <p class="caption" role="heading"><span class="caption-text">Overview</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="01-Home.html">Home</a></li>
<li class="toctree-l1"><a class="reference internal" href="02-Search.html">Search</a></li>
<li class="toctree-l1"><a class="reference internal" href="03-Explorer.html">Explorer</a></li>
<li class="toctree-l1"><a class="reference internal" href="04-Download.html">Download</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">References</a></li>
<li class="toctree-l1"><a class="reference internal" href="06-Document.html">Document</a></li>
<li class="toctree-l1"><a class="reference internal" href="07-About.html">About</a></li>
<li class="toctree-l1"><a class="reference internal" href="08-Case_studies.html">Case Studies</a></li>
<li class="toctree-l1"><a class="reference internal" href="09-Software.html">Software</a></li>
<li class="toctree-l1"><a class="reference internal" href="10-Citation.html">Citation</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">PlantPhoneDB-Tutorial</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="index.html" class="icon icon-home"></a> &raquo;</li>
      <li>References</li>
      <li class="wy-breadcrumbs-aside">
            <a href="_sources/05-References.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="references">
<h1>References<a class="headerlink" href="#references" title="Permalink to this headline">¶</a></h1>
<p>We manually reviewed and confirmed 23 peer-reviewed publications and 4 preprints, and collected the information of 29 scRNA-seq datasets, including ~560,000 cells of 15 tissues from 5 plant species, including <cite>Arabidopsis thaliana</cite>, <cite>Oryza sativa</cite>, <cite>Populus alba x Populus glandulosa</cite>, <cite>Solanum lycopersicum</cite> and <cite>Zea mays</cite>.</p>
<table class="docutils align-default" id="id1">
<caption><span class="caption-text">Table Title</span><a class="headerlink" href="#id1" title="Permalink to this table">¶</a></caption>
<colgroup>
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>PMID</p></th>
<th class="head"><p>Dataset</p></th>
<th class="head"><p>Species</p></th>
<th class="head"><p>Tissue</p></th>
<th class="head"><p>Cells</p></th>
<th class="head"><p>Platform</p></th>
<th class="head"><p>Publication</p></th>
<th class="head"><p>Pre-processing</p></th>
<th class="head"><p>Abstract</p></th>
<th class="head"><p>Title</p></th>
<th class="head"><p>Journal</p></th>
<th class="head"><p>Year</p></th>
<th class="head"><p>Link</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>25608970</p></td>
<td><p>GSE46226</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>31</p></td>
<td><p>Illumina HiSeq</p></td>
<td><p>Efroni I, et al. Genome Biol 2015</p></td>
<td><p>FAIL</p></td>
<td><p>The definition of cell identity is a central problem in biology. While single-cell RNA-seq provides a wealth of information regarding cell states, better methods are needed to map their identity, especially during developmental transitions. Here, we use repositories of cell type-specific transcriptomes to quantify identities from single-cell RNA-seq profiles, accurately classifying cells from Arabidopsis root tips and human glioblastoma tumors. We apply our approach to single cells captured from regenerating roots following tip excision. Our technique exposes a previously uncharacterized transient collapse of identity distant from the injury site, demonstrating the biological relevance of a quantitative cell identity index.</p></td>
<td><p>Quantification of cell identity from single-cell gene expression profiles</p></td>
<td><p>Genome Biol</p></td>
<td><p>2015</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/25608970">https://pubmed.ncbi.nlm.nih.gov/25608970</a></p></td>
</tr>
<tr class="row-odd"><td><p>27212234</p></td>
<td><p>GSE74488</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>215</p></td>
<td><p>Illumina HiSeq</p></td>
<td><p>Efroni I, et al. Cell 2016</p></td>
<td><p>FAIL</p></td>
<td><p>Plant roots can regenerate after excision of their tip, including the stem cell niche. To determine which developmental program mediates such repair, we applied a combination of lineage tracing, single-cell RNA sequencing, and marker analysis to test different models of tissue reassembly. We show that multiple cell types can reconstitute stem cells, demonstrating the latent potential of untreated plant cells. The transcriptome of regenerating cells prior to stem cell activation resembles that of an embryonic root progenitor. Regeneration defects are more severe in embryonic than in adult root mutants. Furthermore, the signaling domains of the hormones auxin and cytokinin mirror their embryonic dynamics and manipulation of both hormones alters the position of new tissues and stem cell niche markers. Our findings suggest that plant root regeneration follows, on a larger scale, the developmental stages of embryonic patterning and is guided by spatial information provided by complementary hormone domains.</p></td>
<td><p>Root Regeneration Triggers an Embryo-like Sequence Guided by Hormonal Interactions</p></td>
<td><p>Cell</p></td>
<td><p>2016</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/27212234">https://pubmed.ncbi.nlm.nih.gov/27212234</a></p></td>
</tr>
<tr class="row-even"><td><p>30718350</p></td>
<td><p>GSE123013</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>11030</p></td>
<td><p>10X Genomics</p></td>
<td><p>Ryu KH, et al. Plant Physiol 2019</p></td>
<td><p>PASS</p></td>
<td><p>Single-cell RNA sequencing (scRNA-seq) has been used extensively to study cell-specific gene expression in animals, but it has not been widely applied to plants. Here, we describe the use of a commercially available droplet-based microfluidics platform for high-throughput scRNA-seq to obtain single-cell transcriptomes from protoplasts of more than 10,000 Arabidopsis ( Arabidopsis thaliana ) root cells. We find that all major tissues and developmental stages are represented in this single-cell transcriptome population. Further, distinct subpopulations and rare cell types, including putative quiescent center cells, were identified. A focused analysis of root epidermal cell transcriptomes defined developmental trajectories for individual cells progressing from meristematic through mature stages of root-hair and nonhair cell differentiation. In addition, single-cell transcriptomes were obtained from root epidermis mutants, enabling a comparative analysis of gene expression at single-cell resolution and providing an unprecedented view of the impact of the mutated genes. Overall, this study demonstrates the feasibility and utility of scRNA-seq in plants and provides a first-generation gene expression map of the Arabidopsis root at single-cell resolution.</p></td>
<td><p>Single-Cell RNA Sequencing Resolves Molecular Relationships Among Individual Plant Cells</p></td>
<td><p>Plant Physiol</p></td>
<td><p>2019</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/30718350">https://pubmed.ncbi.nlm.nih.gov/30718350</a></p></td>
</tr>
<tr class="row-odd"><td><p>30913408</p></td>
<td><p>GSE123818</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>5767</p></td>
<td><p>10X Genomics</p></td>
<td><p>Denyer T, et al. Dev Cell 2019</p></td>
<td><p>PASS</p></td>
<td><p>High-throughput single-cell RNA sequencing (scRNA-seq) is becoming a cornerstone of developmental research, providing unprecedented power in understanding dynamic processes. Here, we present a high-resolution scRNA-seq expression atlas of the Arabidopsis root composed of thousands of independently profiled cells. This atlas provides detailed spatiotemporal information, identifying defining expression features for all major cell types, including the scarce cells of the quiescent center. These reveal key developmental regulators and downstream genes that translate cell fate into distinctive cell shapes and functions. Developmental trajectories derived from pseudotime analysis depict a finely resolved cascade of cell progressions from the niche through differentiation that are supported by mirroring expression waves of highly interconnected transcription factors. This study demonstrates the power of applying scRNA-seq to plants and provides an unparalleled spatiotemporal perspective of root cell differentiation.</p></td>
<td><p>Spatiotemporal Developmental Trajectories in the Arabidopsis Root Revealed Using High-Throughput Single-Cell RNA Sequencing</p></td>
<td><p>Dev Cell</p></td>
<td><p>2019</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/30913408">https://pubmed.ncbi.nlm.nih.gov/30913408</a></p></td>
</tr>
<tr class="row-even"><td><p>30923229</p></td>
<td><p>GSE121619</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>3121</p></td>
<td><p>10X Genomics</p></td>
<td><p>Jean-Baptiste K, et al. Plant Cell 2019</p></td>
<td><p>PASS</p></td>
<td><p>Single cell RNA sequencing can yield high-resolution cell-type-specific expression signatures that reveal new cell types and the developmental trajectories of cell lineages. Here, we apply this approach to Arabidopsis ( Arabidopsis thaliana ) root cells to capture gene expression in 3,121 root cells. We analyze these data with Monocle 3, which orders single cell transcriptomes in an unsupervised manner and uses machine learning to reconstruct single cell developmental trajectories along pseudotime. We identify hundreds of genes with cell-type-specific expression, with pseudotime analysis of several cell lineages revealing both known and novel genes that are expressed along a developmental trajectory. We identify transcription factor motifs that are enriched in early and late cells, together with the corresponding candidate transcription factors that likely drive the observed expression patterns. We assess and interpret changes in total RNA expression along developmental trajectories and show that trajectory branch points mark developmental decisions. Finally, by applying heat stress to whole seedlings, we address the longstanding question of possible heterogeneity among cell types in the response to an abiotic stress. Although the response of canonical heat-shock genes dominates expression across cell types, subtle but significant differences in other genes can be detected among cell types. Taken together, our results demonstrate that single cell transcriptomics holds promise for studying plant development and plant physiology with unprecedented resolution.</p></td>
<td><p>Dynamics of Gene Expression in Single Root Cells of Arabidopsis thaliana</p></td>
<td><p>Plant Cell</p></td>
<td><p>2019</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/30923229">https://pubmed.ncbi.nlm.nih.gov/30923229</a></p></td>
</tr>
<tr class="row-odd"><td><p>30948545</p></td>
<td><p>GSE121039</p></td>
<td><p>Zea mays</p></td>
<td><p>Anther</p></td>
<td><p>213</p></td>
<td><p>CelSeq2</p></td>
<td><p>Nelms B, et al. Science 2019</p></td>
<td><p>FAIL</p></td>
<td><p>In multicellular organisms, the entry into meiosis is a complex process characterized by increasing meiotic specialization. Using single-cell RNA sequencing, we reconstructed the developmental program into maize male meiosis. A smooth continuum of expression stages before meiosis was followed by a two-step transcriptome reorganization in leptotene, during which 26.7% of transcripts changed in abundance by twofold or more. Analysis of cell-cycle gene expression indicated that nearly all pregerminal cells proliferate, eliminating a stem-cell model to generate meiotic cells. Mutants defective in somatic differentiation or meiotic commitment expressed transcripts normally present in early meiosis after a delay; thus, the germinal transcriptional program is cell autonomous and can proceed despite meiotic failure.</p></td>
<td><p>Defining the developmental program leading to meiosis in maize</p></td>
<td><p>Science</p></td>
<td><p>2019</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/30948545">https://pubmed.ncbi.nlm.nih.gov/30948545</a></p></td>
</tr>
<tr class="row-even"><td><p>31004836</p></td>
<td><p>PRJNA517021</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>~15,000</p></td>
<td><p>10X Genomics</p></td>
<td><p>Tian-Qi Zhang, et al. Mol Plant 2019</p></td>
<td><p>pending</p></td>
<td><p>Cells of eukaryotic multicellular organisms have inherent heterogeneity. Recent advances in single-cell gene expression studies enable us to explore transcriptional regulation in dynamic development processes and highly heterogeneous cell populations. In this study, using a high-throughput single-cell RNA-sequencing assay, we found that the cells in Arabidopsis root are highly heterogeneous in their transcriptomes. A total of 24 putative cell clusters and the cluster-specific marker genes were identified. The spatial distribution and temporal ordering of the individual cells at different developmental stages illustrate their hierarchical structures and enable the reconstruction of continuous differentiation trajectory of root development. Moreover, we found that each root cell cluster exhibits distinct patterns of ion assimilation and hormonal responses. Collectively, our study reveals a high degree of heterogeneity of root cells and identifies the expression signatures of intermediate states during root cell differentiation at single-cell resolution. We also established a web server (<a class="reference external" href="http://wanglab.sippe.ac.cn/rootatlas/">http://wanglab.sippe.ac.cn/rootatlas/</a>) to facilitate the use of the datasets generated in this study.</p></td>
<td><p>A Single-Cell RNA Sequencing Profiles the Developmental Landscape of Arabidopsis Root</p></td>
<td><p>Mol Plant</p></td>
<td><p>2019</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/31004836">https://pubmed.ncbi.nlm.nih.gov/31004836</a></p></td>
</tr>
<tr class="row-odd"><td><p>31091459</p></td>
<td><p>GSE122687</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>12198</p></td>
<td><p>Drop-seq</p></td>
<td><p>Shulse CN, et al. Cell Rep 2019</p></td>
<td><p>PASS</p></td>
<td><p>Single-cell transcriptome profiling of heterogeneous tissues can provide high-resolution windows into developmental dynamics and environmental responses, but its application to plants has been limited. Here, we used the high-throughput Drop-seq approach to profile &gt;12,000 cells from Arabidopsis roots. This identified numerous distinct cell types, covering all major root tissues and developmental stages, and illuminated specific marker genes for these populations. In addition, we demonstrate the utility of this approach to study the impact of environmental conditions on developmental processes. Analysis of roots grown with or without sucrose supplementation uncovers changes in the relative frequencies of cell types in response to sucrose. Finally, we characterize the transcriptome changes that occur across endodermis development and identify nearly 800 genes with dynamic expression as this tissue differentiates. Collectively, we demonstrate that single-cell RNA-seq can be used to profile developmental processes in plants and show how they can be altered by external stimuli.</p></td>
<td><p>High-Throughput Single-Cell Transcriptome Profiling of Plant Cell Types</p></td>
<td><p>Cell Rep</p></td>
<td><p>2019</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/31091459">https://pubmed.ncbi.nlm.nih.gov/31091459</a></p></td>
</tr>
<tr class="row-even"><td><p>31291572</p></td>
<td><p>GSE114615</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>3897</p></td>
<td><p>Drop-seq</p></td>
<td><p>Turco GM, et al. Cell Rep 2019</p></td>
<td><p>PASS</p></td>
<td><p>Plant xylem cells conduct water and mineral nutrients. Although most plant cells are totipotent, xylem cells are unusual and undergo terminal differentiation. Many genes regulating this process are well characterized, including the Vascular-related NAC Domain 7 (VND7), MYB46, and MYB83 transcription factors, which are proposed to act in interconnected feedforward loops (FFLs). Less is known regarding the molecular mechanisms underlying the terminal transition to xylem cell differentiation. Here, we generate whole-root and single-cell data, which demonstrate that VND7 initiates sharp switching of root cells to xylem cell identity. Based on these data, we identified 4 candidate VND7 downstream target genes capable of generating this switch. Although MYB46 responds to VND7 induction, it is not among these targets. This system provides an important model to study the emergent properties that may give rise to totipotency relative to terminal differentiation and reveals xylem cell subtypes.</p></td>
<td><p>Molecular Mechanisms Driving Switch Behavior in Xylem Cell Differentiation</p></td>
<td><p>Cell Rep</p></td>
<td><p>2019</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/31291572">https://pubmed.ncbi.nlm.nih.gov/31291572</a></p></td>
</tr>
<tr class="row-odd"><td><p>32592820</p></td>
<td><p>PRJNA577177</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>cotyledon</p></td>
<td><p>12844</p></td>
<td><p>10x Genomics</p></td>
<td><p>Liu, et al. Mol Plant 2020</p></td>
<td><p>pending</p></td>
<td><p>The regulation of stomatal lineage cell development has been extensively investigated. However, a comprehensive characterization of this biological process based on single-cell transcriptome analysis has not yet been reported. In this study, we performed RNA sequencing on 12,844 individual cells from the cotyledons of 5-day-old Arabidopsis seedlings. We identified 11 cell clusters corresponding mostly to cells at specific stomatal developmental stages using a series of marker genes. Comparative analysis of genes with the highest variable expression among these cell clusters revealed transcriptional networks that regulate development from meristemoid mother cells to guard mother cells. Examination of the developmental dynamics of marker genes via pseudo-time analysis revealed potential interactions between these genes. Collectively, our study opens the door for understanding how the identified novel marker genes participate in the regulation of stomatal lineage cell development.</p></td>
<td><p>Global Dynamic Molecular Profiling of Stomatal Lineage Cell Development by Single-Cell RNA Sequencing</p></td>
<td><p>Mol Plant</p></td>
<td><p>2020</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/32592820">https://pubmed.ncbi.nlm.nih.gov/32592820</a></p></td>
</tr>
<tr class="row-even"><td><p>32943451</p></td>
<td><p>GSE141730</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>15918</p></td>
<td><p>10X Genomics</p></td>
<td><p>Wendrich JR, et al. Science 2020</p></td>
<td><p>PASS</p></td>
<td><p>Optimal plant growth is hampered by deficiency of the essential macronutrient phosphate in most soils. Plant roots can, however, increase their root hair density to efficiently forage the soil for this immobile nutrient. By generating and exploiting a high-resolution single-cell gene expression atlas of  Arabidopsis  roots, we show an enrichment of TARGET OF MONOPTEROS 5/LONESOME HIGHWAY (TMO5/LHW) target gene responses in root hair cells. The TMO5/LHW heterodimer triggers biosynthesis of mobile cytokinin in vascular cells and increases root hair density during low-phosphate conditions by modifying both the length and cell fate of epidermal cells. Moreover, root hair responses in phosphate-deprived conditions are TMO5- and cytokinin-dependent. Cytokinin signaling links root hair responses in the epidermis to perception of phosphate depletion in vascular cells.</p></td>
<td><p>Vascular transcription factors guide plant epidermal responses to limiting phosphate conditions</p></td>
<td><p>Science</p></td>
<td><p>2020</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/32943451">https://pubmed.ncbi.nlm.nih.gov/32943451</a></p></td>
</tr>
<tr class="row-odd"><td><p>33318187</p></td>
<td><p>PRJNA637882</p></td>
<td><p>Zea mays</p></td>
<td><p>Shoot apex</p></td>
<td><p>8089</p></td>
<td><p>10X Genomics</p></td>
<td><p>James W, et al. PNAS 2020</p></td>
<td><p>FAIL</p></td>
<td><p>Plants maintain populations of pluripotent stem cells in shoot apical meristems (SAMs), which continuously produce new aboveground organs. We used single-cell RNA sequencing (scRNA-seq) to achieve an unbiased characterization of the transcriptional landscape of the maize shoot stem-cell niche and its differentiating cellular descendants. Stem cells housed in the SAM tip are engaged in genome integrity maintenance and exhibit a low rate of cell division, consistent with their contributions to germline and somatic cell fates. Surprisingly, we find no evidence for a canonical stem-cell organizing center subtending these cells. In addition, trajectory inference was used to trace the gene expression changes that accompany cell differentiation, revealing that ectopic expression of  KNOTTED1  ( KN1 ) accelerates cell differentiation and promotes development of the sheathing maize leaf base. These single-cell transcriptomic analyses of the shoot apex yield insight into the processes of stem-cell function and cell-fate acquisition in the maize seedling and provide a valuable scaffold on which to better dissect the genetic control of plant shoot morphogenesis at the cellular level.</p></td>
<td><p>Plant stem-cell organization and differentiation at single-cell resolution</p></td>
<td><p>Proc Natl Acad Sci U S A</p></td>
<td><p>2020</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33318187">https://pubmed.ncbi.nlm.nih.gov/33318187</a></p></td>
</tr>
<tr class="row-even"><td><p>33352304</p></td>
<td><p>GSE146035</p></td>
<td><p>Oryza sativa</p></td>
<td><p>Root tips</p></td>
<td><p>23532</p></td>
<td><p>10X Genomics</p></td>
<td><p>Liu Q, et al. Mol Plant 2021</p></td>
<td><p>PASS</p></td>
<td><p>There are two main types of root systems in flowering plants, namely taproot systems of dicots and fibrous root systems found in monocots. Despite this fundamental split, our current knowledge of cellular and molecular mechanism driving root development is mainly based on studies of the dicot model Arabidopsis. However, the world major crops are monocots and little is known about the transcriptional programs underlying cell-type specification in this clade. Here, we report the transcriptomes of more than 20,000 single cells derived from root tips of two agronomically important rice cultivars. Using combined computational and experimental analyses we were able to robustly identify most of the major cell types and define novel cell-type-specific marker genes for both cultivars. Importantly, we found divergent cell types associated with specific regulatory programs, including phytohormone biosynthesis, signaling, and response, which were well conserved between the two rice cultivars. In addition, we detected substantial differences between the cell-type transcript profiles of Arabidopsis and rice. These species-specific features emphasize the importance of analyzing tissues across diverse model species, including rice. Taken together, our study provides insight into the transcriptomic landscape of major cell types of rice root tip at single-cell resolution and opens new avenues to study cell-type specification, function, and evolution in plants.</p></td>
<td><p>Transcriptional landscape of rice roots at the single-cell resolution</p></td>
<td><p>Mol Plant</p></td>
<td><p>2020</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33352304">https://pubmed.ncbi.nlm.nih.gov/33352304</a></p></td>
</tr>
<tr class="row-odd"><td><p>33400914</p></td>
<td><p>PRJNA646989_PRJNA646996_PRJNA647001</p></td>
<td><p>Zea mays</p></td>
<td><p>Ears</p></td>
<td><p>12525</p></td>
<td><p>10X Genomics</p></td>
<td><p>Xiaosa Xu, et al. Dev Cell 2021</p></td>
<td><p>PASS</p></td>
<td><p>Crop productivity depends on activity of meristems that produce optimized plant architectures, including that of the maize ear. A comprehensive understanding of development requires insight into the full diversity of cell types and developmental domains and the gene networks required to specify them. Until now, these were identified primarily by morphology and insights from classical genetics, which are limited by genetic redundancy and pleiotropy. Here, we investigated the transcriptional profiles of 12,525 single cells from developing maize ears. The resulting developmental atlas provides a single-cell RNA sequencing (scRNA-seq) map of an inflorescence. We validated our results by mRNA in situ hybridization and by fluorescence-activated cell sorting (FACS) RNA-seq, and we show how these data may facilitate genetic studies by predicting genetic redundancy, integrating transcriptional networks, and identifying candidate genes associated with crop yield traits.</p></td>
<td><p>Single-cell RNA sequencing of developing maize ears facilitates functional analysis and trait candidate gene discovery</p></td>
<td><p>Dev Cell</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33400914">https://pubmed.ncbi.nlm.nih.gov/33400914</a></p></td>
</tr>
<tr class="row-even"><td><p>33422696</p></td>
<td><p>GSE155304</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>10548</p></td>
<td><p>10X Genomics</p></td>
<td><p>Farmer A, et al. Mol Plant 2021</p></td>
<td><p>PASS</p></td>
<td><p>Similar to other complex organisms, plants consist of diverse and specialized cell types. The gain of unique biological functions of these different cell types is the consequence of the establishment of cell-type-specific transcriptional programs. As a necessary step in gaining a deeper understanding of the regulatory mechanisms controlling plant gene expression, we report the use of single-nucleus RNA sequencing (sNucRNA-seq) and single-nucleus assay for transposase accessible chromatin sequencing (sNucATAC-seq) technologies on Arabidopsis roots. The comparison of our single-nucleus transcriptomes to the published protoplast transcriptomes validated the use of nuclei as biological entities to establish plant cell-type-specific transcriptomes. Furthermore, our sNucRNA-seq results uncovered the transcriptomes of additional cell subtypes not identified by single-cell RNA-seq. Similar to our transcriptomic approach, the sNucATAC-seq approach led to the distribution of the Arabidopsis nuclei into distinct clusters, suggesting the differential accessibility of chromatin between groups of cells according to their identity. To reveal the impact of chromatin accessibility on gene expression, we integrated sNucRNA-seq and sNucATAC-seq data and demonstrated that cell-type-specific marker genes display cell-type-specific patterns of chromatin accessibility. Our data suggest that the differential chromatin accessibility is a critical mechanism to regulate gene activity at the cell-type level.</p></td>
<td><p>Single-nucleus RNA and ATAC sequencing reveals the impact of chromatin accessibility on gene expression in Arabidopsis roots at the single-cell level</p></td>
<td><p>Mol Plant</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33422696">https://pubmed.ncbi.nlm.nih.gov/33422696</a></p></td>
</tr>
<tr class="row-odd"><td><p>33608047</p></td>
<td><p>PRJNA664874</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>1186</p></td>
<td><p>10x Genomics</p></td>
<td><p>Yanping Long, et al. Genome Biol 2021</p></td>
<td><p>pending</p></td>
<td><p>The broad application of single-cell RNA profiling in plants has been hindered by the prerequisite of protoplasting that requires digesting the cell walls from different types of plant tissues. Here, we present a protoplasting-free approach, flsnRNA-seq, for large-scale full-length RNA profiling at a single-nucleus level in plants using isolated nuclei. Combined with 10x Genomics and Nanopore long-read sequencing, we validate the robustness of this approach in Arabidopsis root cells and the developing endosperm. Sequencing results demonstrate that it allows for uncovering alternative splicing and polyadenylation-related RNA isoform information at the single-cell level, which facilitates characterizing cell identities.</p></td>
<td><p>FlsnRNA-seq: protoplasting-free full-length single-nucleus RNA profiling in plants</p></td>
<td><p>Genome Biol</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33608047">https://pubmed.ncbi.nlm.nih.gov/33608047</a></p></td>
</tr>
<tr class="row-even"><td><p>33608047</p></td>
<td><p>PRJNA685588</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Endosperm</p></td>
<td><p>563</p></td>
<td><p>10x Genomics</p></td>
<td><p>Yanping Long, et al. Genome Biol 2021</p></td>
<td><p>pending</p></td>
<td><p>The broad application of single-cell RNA profiling in plants has been hindered by the prerequisite of protoplasting that requires digesting the cell walls from different types of plant tissues. Here, we present a protoplasting-free approach, flsnRNA-seq, for large-scale full-length RNA profiling at a single-nucleus level in plants using isolated nuclei. Combined with 10x Genomics and Nanopore long-read sequencing, we validate the robustness of this approach in Arabidopsis root cells and the developing endosperm. Sequencing results demonstrate that it allows for uncovering alternative splicing and polyadenylation-related RNA isoform information at the single-cell level, which facilitates characterizing cell identities.</p></td>
<td><p>FlsnRNA-seq: protoplasting-free full-length single-nucleus RNA profiling in plants</p></td>
<td><p>Genome Biol</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33608047">https://pubmed.ncbi.nlm.nih.gov/33608047</a></p></td>
</tr>
<tr class="row-odd"><td><p>33822225</p></td>
<td><p>GSE158761</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Lateral root</p></td>
<td><p>6658</p></td>
<td><p>10X Genomics</p></td>
<td><p>Gala HP, et al. Plant Cell 2021</p></td>
<td><p>PASS</p></td>
<td><p>Root architecture is a major determinant of plant fitness and is under constant modification in response to favorable and unfavorable environmental stimuli. Beyond impacts on the primary root, the environment can alter the position, spacing, density, and length of secondary or lateral roots. Lateral root development is among the best-studied examples of plant organogenesis, yet there are still many unanswered questions about its earliest steps. Among the challenges faced in capturing these first molecular events is the fact that this process occurs in a small number of cells with unpredictable timing. Single-cell sequencing methods afford the opportunity to isolate the specific transcriptional changes occurring in cells undergoing this fate transition. Using this approach, we successfully captured the transcriptomes of initiating lateral root primordia in Arabidopsis thaliana and discovered many upregulated genes associated with this process. We developed a method to selectively repress target gene transcription in the xylem pole pericycle cells where lateral roots originate and demonstrated that the expression of several of these targets is required for normal root development. We also discovered subpopulations of cells in the pericycle and endodermal cell files that respond to lateral root initiation, highlighting the coordination across cell files required for this fate transition.</p></td>
<td><p>A single cell view of the transcriptome during lateral root initiation in Arabidopsis thaliana</p></td>
<td><p>Plant Cell</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33822225">https://pubmed.ncbi.nlm.nih.gov/33822225</a></p></td>
</tr>
<tr class="row-even"><td><p>33823130</p></td>
<td><p>GSE167135</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Leaf</p></td>
<td><p>18000</p></td>
<td><p>10x Genomics &amp; Smart-seq2</p></td>
<td><p>Lopez-Anido CB, et al. Dev Cell  2021</p></td>
<td><p>pending</p></td>
<td><p>Dynamic cell identities underlie flexible developmental programs. The stomatal lineage in the Arabidopsis leaf epidermis features asynchronous and indeterminate divisions that can be modulated by environmental cues. The products of the lineage, stomatal guard cells and pavement cells, regulate plant-atmosphere exchanges, and the epidermis as a whole influences overall leaf growth. How flexibility is encoded in development of the stomatal lineage and how cell fates are coordinated in the leaf are open questions. Here, by leveraging single-cell transcriptomics and molecular genetics, we uncovered models of cell differentiation within Arabidopsis leaf tissue. Profiles across leaf tissues identified points of regulatory congruence. In the stomatal lineage, single-cell resolution resolved underlying cell heterogeneity within early stages and provided a fine-grained profile of guard cell differentiation. Through integration of genome-scale datasets and spatiotemporally precise functional manipulations, we also identified an extended role for the transcriptional regulator SPEECHLESS in reinforcing cell fate commitment.</p></td>
<td><p>Single-cell resolution of lineage trajectories in the Arabidopsis stomatal lineage and developing leaf</p></td>
<td><p>Dev Cell</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33823130">https://pubmed.ncbi.nlm.nih.gov/33823130</a></p></td>
</tr>
<tr class="row-odd"><td><p>33955487</p></td>
<td><p>GSE161482</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Leaf</p></td>
<td><p>5230</p></td>
<td><p>10X Genomics</p></td>
<td><p>Kim JY, et al. Plant Cell 2021</p></td>
<td><p>PASS</p></td>
<td><p>The leaf vasculature plays a key role in solute translocation. Veins consist of at least seven distinct cell types, with specific roles in transport, metabolism, and signaling. Little is known about leaf vascular cells, in particular the phloem parenchyma (PP). PP effluxes sucrose into the apoplasm as a basis for phloem loading, yet PP has been characterized only microscopically. Here, we enriched vascular cells from Arabidopsis leaves to generate a single-cell transcriptome atlas of leaf vasculature. We identified at least 19 cell clusters, encompassing epidermis, guard cells, hydathodes, mesophyll, and all vascular cell types, and used metabolic pathway analysis to define their roles. Clusters comprising PP cells were enriched for transporters, including SWEET11 and SWEET12 sucrose and UmamiT amino acid efflux carriers. We provide evidence that PP development occurs independently from ALTERED PHLOEM DEVELOPMENT, a transcription factor required for phloem differentiation. PP cells have a unique pattern of amino acid metabolism activity distinct from companion cells (CCs), explaining differential distribution/metabolism of amino acids in veins. The kinship relation of the vascular clusters is strikingly similar to the vein morphology, except for a clear separation of CC from the other vascular cells including PP. In summary, our single-cell RNA-sequencing analysis provides a wide range of information into the leaf vasculature and the role and relationship of the leaf cell types.</p></td>
<td><p>Distinct identities of leaf phloem cells revealed by single cell transcriptomics</p></td>
<td><p>Plant Cell</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33955487">https://pubmed.ncbi.nlm.nih.gov/33955487</a></p></td>
</tr>
<tr class="row-even"><td><p>33955487</p></td>
<td><p>GSE161332</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Leaf</p></td>
<td><p>6300</p></td>
<td><p>10X Genomics</p></td>
<td><p>Kim JY, et al. Plant Cell 2021</p></td>
<td><p>PASS</p></td>
<td><p>The leaf vasculature plays a key role in solute translocation. Veins consist of at least seven distinct cell types, with specific roles in transport, metabolism, and signaling. Little is known about leaf vascular cells, in particular the phloem parenchyma (PP). PP effluxes sucrose into the apoplasm as a basis for phloem loading, yet PP has been characterized only microscopically. Here, we enriched vascular cells from Arabidopsis leaves to generate a single-cell transcriptome atlas of leaf vasculature. We identified at least 19 cell clusters, encompassing epidermis, guard cells, hydathodes, mesophyll, and all vascular cell types, and used metabolic pathway analysis to define their roles. Clusters comprising PP cells were enriched for transporters, including SWEET11 and SWEET12 sucrose and UmamiT amino acid efflux carriers. We provide evidence that PP development occurs independently from ALTERED PHLOEM DEVELOPMENT, a transcription factor required for phloem differentiation. PP cells have a unique pattern of amino acid metabolism activity distinct from companion cells (CCs), explaining differential distribution/metabolism of amino acids in veins. The kinship relation of the vascular clusters is strikingly similar to the vein morphology, except for a clear separation of CC from the other vascular cells including PP. In summary, our single-cell RNA-sequencing analysis provides a wide range of information into the leaf vasculature and the role and relationship of the leaf cell types.</p></td>
<td><p>Distinct identities of leaf phloem cells revealed by single cell transcriptomics</p></td>
<td><p>Plant Cell</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33955487">https://pubmed.ncbi.nlm.nih.gov/33955487</a></p></td>
</tr>
<tr class="row-odd"><td><p>33955497</p></td>
<td><p>GSE157757</p></td>
<td><p>Zea mays</p></td>
<td><p>Leaf</p></td>
<td><p>3763</p></td>
<td><p>10x Genomics</p></td>
<td><p>Bezrutczyk M, et al. Plant Cell 2021</p></td>
<td><p>PASS</p></td>
<td><p>Leaves are asymmetric, with different functions for adaxial and abaxial tissue. The bundle sheath (BS) of C3 barley (Hordeum vulgare) is dorsoventrally differentiated into three types of cells: adaxial structural, lateral S-type, and abaxial L-type BS cells. Based on plasmodesmatal connections between S-type cells and mestome sheath (parenchymatous cell layer below bundle sheath), S-type cells likely transfer assimilates toward the phloem. Here, we used single-cell RNA sequencing to investigate BS differentiation in C4 maize (Zea mays L.) plants. Abaxial BS (abBS) cells of rank-2 intermediate veins specifically expressed three SWEET sucrose uniporters (SWEET13a, b, and c) and UmamiT amino acid efflux transporters. SWEET13a, b, c mRNAs were also detected in the phloem parenchyma (PP). We show that maize has acquired a mechanism for phloem loading in which abBS cells provide the main route for apoplasmic sucrose transfer toward the phloem. This putative route predominates in veins responsible for phloem loading (rank-2 intermediate), whereas rank-1 intermediate and major veins export sucrose from the PP adjacent to the sieve element companion cell complex, as in Arabidopsis thaliana. We surmise that abBS identity is subject to dorsoventral patterning and has components of PP identity. These observations provide insights into the unique transport-specific properties of abBS cells and support a modification to the canonical phloem loading pathway in maize.</p></td>
<td><p>Evidence for phloem loading via the abaxial bundle sheath cells in maize leaves</p></td>
<td><p>Plant Cell</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33955497">https://pubmed.ncbi.nlm.nih.gov/33955497</a></p></td>
</tr>
<tr class="row-even"><td><p>34062316</p></td>
<td><p>GSE161970</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Lateral root</p></td>
<td><p>573</p></td>
<td></td>
<td><p>Serrano-Ron L, et al. Mol Plant 2021</p></td>
<td><p>FAIL</p></td>
<td><p>Postembryonic organogenesis is critical for plant development. Underground, lateral roots (LRs) form the bulk of mature root systems, yet the ontogeny of the LR primordium (LRP) is not clear. In this study, we performed the single-cell RNA sequencing through the first four stages of LR formation in Arabidopsis. Our analysis led to a model in which a single group of precursor cells, with a cell identity different from their pericycle origins, rapidly reprograms and splits into a mixed ground tissue/stem cell niche fate and a vascular precursor fate. The ground tissue and stem cell niche fates soon separate and a subset of more specialized vascular cells form sucrose transporting phloem cells that appear to connect to the primary root. We did not detect cells resembling epidermis or root cap, suggesting that outer tissues may form later, preceding LR emergence. At this stage, some remaining initial precursor cells form the primordium flanks, while the rest create a reservoir of pluripotent cells that are able to replace the LR if damaged. Laser ablation of the central and lateral LRP regions showed that remaining cells restart the sequence of tissue initiation to form a LR. Collectively, our study reveals an ontological hierarchy for LR formation with an early and sequential split of main root tissues and stem cells.</p></td>
<td><p>Reconstruction of lateral root formation through single-cell RNA sequencing reveals order of tissue initiation</p></td>
<td><p>Mol Plant</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/34062316">https://pubmed.ncbi.nlm.nih.gov/34062316</a></p></td>
</tr>
<tr class="row-odd"><td><p>34340913</p></td>
<td><p>CRA004082</p></td>
<td><p>Oryza sativa</p></td>
<td><p>root;leaf</p></td>
<td><p>237431</p></td>
<td><p>10x Genomics</p></td>
<td><p>Yu Wang, et al.  J Genet Genomics 2021</p></td>
<td><p>pending</p></td>
<td><p>As a multicellular organism, rice flourishes relying on gene expression diversity among cells of various functions. However, cellular-resolution transcriptome features are yet to be fully recognized, let alone cell-specific transcriptional responses to environmental stimuli. In this study, we apply single-cell RNA sequencing to both shoot and root of rice seedlings growing in Kimura B nutrient solution or exposed to various abiotic stresses and characterize transcriptomes for a total of 237,431 individual cells. We identify 15 and nine cell types in the leaf and root, respectively, and observe that common transcriptome features are often shared between leaves and roots in the same tissue layer, except for endodermis or epidermis. Abiotic stress stimuli alter gene expression largely in a cell type-specific manner, but for a given cell type, different stresses often trigger transcriptional regulation of roughly the same set of genes. Besides, we detect proportional changes in cell populations in response to abiotic stress and investigate the underlying molecular mechanisms through single-cell reconstruction of the developmental trajectory. Collectively, our study represents a benchmark-setting data resource of single-cell transcriptome atlas for rice seedlings and an illustration of exploiting such resources to drive discoveries in plant biology.</p></td>
<td><p>Single-cell transcriptome atlas of the leaf and root of rice seedlings</p></td>
<td><p>J Genet Genomics</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/34340913">https://pubmed.ncbi.nlm.nih.gov/34340913</a></p></td>
</tr>
<tr class="row-even"><td><p>34347368</p></td>
<td><p>PRJNA703312</p></td>
<td><p>Populus alba x Populus glandulosa</p></td>
<td><p>Xylem</p></td>
<td><p>10646</p></td>
<td><p>Drop-seq</p></td>
<td><p>Hui Li, et al.  J Integr Plant Biol 2021</p></td>
<td><p>pending</p></td>
<td><p>High-throughput single-cell RNA sequencing (scRNA-seq) has advantages over traditional RNA-seq to explore spatiotemporal information on gene dynamic expressions in heterogenous tissues. We performed Drop-seq, a method for the dropwise sequestration of single cells for sequencing, on protoplasts from the differentiating xylem of Populus alba × Populus glandulosa. The scRNA-seq profiled 9,798 cells, which were grouped into 12 clusters. Through characterization of differentially expressed genes in each cluster and RNA in situ hybridizations, we identified vessel cells, fiber cells, ray parenchyma cells and xylem precursor cells. Diffusion pseudotime analyses revealed the differentiating trajectory of vessels, fiber cells and ray parenchyma cells and indicated a different differentiation process between vessels and fiber cells, and a similar differentiation process between fiber cells and ray parenchyma cells. We identified marker genes for each cell type (cluster) and key candidate regulators during developmental stages of xylem cell differentiation. Our study generates a high resolution expression atlas of wood formation at the single cell level and provides valuable information on wood formation.</p></td>
<td><p>Single-cell RNA sequencing reveals a high-resolution cell atlas of xylem in Populus</p></td>
<td><p>J Integr Plant Biol</p></td>
<td><p>2021</p></td>
<td><p><a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/34347368">https://pubmed.ncbi.nlm.nih.gov/34347368</a></p></td>
</tr>
<tr class="row-odd"><td></td>
<td><p>GSE152766</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>Root</p></td>
<td><p>110000</p></td>
<td><p>10x Genomics</p></td>
<td><p>Shahan R, et al. bioRxiv 2020</p></td>
<td><p>pending</p></td>
<td><p>Cell fate acquisition is a fundamental developmental process in all multicellular organisms. Yet, much is unknown regarding how a cell traverses the pathway from stem cell to terminal differentiation. Advances in single cell genomics hold promise for unraveling developmental mechanisms in tissues, organs, and organisms. However, lineage tracing can be challenging for some tissues and integration of high-quality datasets is often necessary to detect rare cell populations and developmental states. Here, we harmonized single cell mRNA sequencing data from over 110,000 cells to construct a comprehensive atlas for a stereotypically developing organ with indeterminate growth, the Arabidopsis root. To test the utility of the atlas to interpret new datasets, we profiled mutants for two key transcriptional regulators at single cell resolution, shortroot and scarecrow. Although both transcription factors are required for early specification of cell identity, our results suggest the existence of an alternative pathway acting in mature cells to specify endodermal identity, for which SHORTROOT is required. Uncovering the architecture of this pathway will provide insight into specification and stabilization of the endodermis, a tissue analogous to the mammalian epithelium. Thus, the atlas is a pivotal advance for unraveling the transcriptional programs that specify and maintain cell identity to regulate organ development in space and time.</p></td>
<td><p>A single cell Arabidopsis root atlas reveals developmental trajectories in wild type and cell identity mutants</p></td>
<td><p>bioRxiv</p></td>
<td><p>2020</p></td>
<td><p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2020.06.29.178863">https://www.biorxiv.org/content/10.1101/2020.06.29.178863</a></p></td>
</tr>
<tr class="row-even"><td></td>
<td><p>SAMN16069893</p></td>
<td><p>Solanum lycopersicum</p></td>
<td><p>shoot apex</p></td>
<td><p>13377</p></td>
<td><p>10x Genomics</p></td>
<td><p>Tian, et al. bioRxiv 2020</p></td>
<td><p>pending</p></td>
<td><p>Single cell transcriptomics is revolutionizing our understanding of development and response to environmental cues. Recent advances in single cell RNA sequencing (scRNA-seq) technology have enabled profiling gene expression pattern of heterogenous tissues and organs at single cellular level and have been widely applied in human and animal research. Nevertheless, the existence of cell walls significantly encumbered its application in plant research. Protoplasts have been applied for scRNA-seq analysis, but mostly restricted to tissues amenable for wall digestion, such as root tips. However, many cell types are resistant to protoplasting, and protoplasting may yield ectopic gene expression and bias proportions of cell types. Here we demonstrate a method with minimal artifacts for high-throughput single-nucleus RNA sequencing (snRNA-Seq) that we use to profile tomato shoot apex cells. The obtained high-resolution expression atlas identifies numerous distinct cell types covering major shoot tissues and developmental stages, delineates developmental trajectories of mesophyll cells, vasculature cells, epidermal cells, and trichome cells. In addition, we identify key developmental regulators and reveal their hierarchy. Collectively, this study demonstrates the power of snRNA-seq to plant research and provides an unprecedented spatiotemporal gene expression atlas of heterogeneous shoot cells.</p></td>
<td><p>Single-nucleus RNA-seq resolves spatiotemporal developmental trajectories in the tomato shoot apex</p></td>
<td><p>bioRxiv</p></td>
<td><p>2020</p></td>
<td><p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2020.09.20.305029v1">https://www.biorxiv.org/content/10.1101/2020.09.20.305029v1</a></p></td>
</tr>
<tr class="row-odd"><td></td>
<td></td>
<td><p>Oryza sativa</p></td>
<td><p>the aerial part of rice seedlings</p></td>
<td><p>4580</p></td>
<td><p>10x Genomics</p></td>
<td><p>Wang, et al. bioRxiv 2020</p></td>
<td><p>pending</p></td>
<td><p>The metabolism and reproduction of plants depend on the division of labors among cells. However, cells with various functions are often studied as a bulk where their specificities were diluted. Here, we apply single-cell RNA sequencing to the aerial part of rice seedlings growing in various environments. We capture the transcriptomes of thousands of cells, identify all major cell types, and reconstruct their developmental trajectories. We find that abiotic stresses not only affect gene expression in a cell-type-specific manner but also have impacts on the physical size of cells and the composition of cell populations. We validate some of these conclusions with microscopy and provide developmental mechanisms with computational analyses. Collectively, our study represents a benchmark-setting data resource of single-cell transcriptome atlas in rice and an illustration of exploiting such resource to drive discoveries in plant biology.</p></td>
<td><p>Single-cell transcriptome analyses recapitulate the cellular and developmental responses to abiotic stresses in rice</p></td>
<td><p>bioRxiv</p></td>
<td><p>2020</p></td>
<td><p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2020.01.30.926329">https://www.biorxiv.org/content/10.1101/2020.01.30.926329</a></p></td>
</tr>
<tr class="row-even"><td></td>
<td><p>E-MTAB-9174</p></td>
<td><p>Arabidopsis thaliana</p></td>
<td><p>root;leaf;flower;seed;silique;stem</p></td>
<td><p>3727</p></td>
<td><p>Nanowell-based approach</p></td>
<td><p>Sunaga-Franze, et al. bioRxiv 2020</p></td>
<td><p>pending</p></td>
<td><p>Single-cell genomics has slowly advanced in plant research. Here, we introduce a generic procedure for plant nuclei isolation and nanowell-based library preparation for short-read sequencing. This plant-nuclei sequencing (PN-seq) method allows for analyzing several thousands of genes in thousands of individual plant cells. In this way, we expand the toolset for single-cell genomics in the field of plant biology to generate plant transcriptome atlases in development and environmental response.</p></td>
<td><p>Single-nuclei RNA-sequencing of plants</p></td>
<td><p>bioRxiv</p></td>
<td><p>2020</p></td>
<td><p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2020.11.14.382812">https://www.biorxiv.org/content/10.1101/2020.11.14.382812</a></p></td>
</tr>
</tbody>
</table>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="04-Download.html" class="btn btn-neutral float-left" title="Download" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="06-Document.html" class="btn btn-neutral float-right" title="Document" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2022, Chaoqun Xu.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>